modules/sample-audit-log-message.adoc

```diff
@@ -58,3 +63,20 @@ The log message has the following format:
   }
 }
 ----
+
+The source IP address of the request is displayed in the source fields, which makes it easier for you to investigate audit log requests and identify their origin.
+
+To determine the source IP address of a request, {product-title-short} uses the following:
```

Comment:
In the context of APIs, I'd be inclined to go with the term _parameters_ instead of fields. Whichever, you use, be consistent with the use across the section.